nexus/nexus.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM sonatype/nexus3:3.76.0-ubi
+
+ARG ADMIN_PASS=admin123
+ENV ADMIN_PASS=$ADMIN_PASS
+COPY init_and_run.sh ${SONATYPE_DIR}/init_and_run.sh
+
+EXPOSE 8081
+EXPOSE 8181
+USER nexus
+
+CMD ["/opt/sonatype/init_and_run.sh"]
```

Comment:
IIRC this gets executed on container startup, right? Just asking because otherwise, Nexus would initialize the admin password and write on disk, and it would be included in the produced image.

Reply:
yes, the `RUN` would be to do something during the build and include in the image, and `CMD` is for when starting.